setup.cfg

```diff
@@ -45,3 +45,6 @@ skip=
     envs
     migrations
     common/lib/safe_lxml/safe_lxml/etree.py
+    wsgi.py
+    wsgi_apache_lms.py
```

Comment:
There are files in this pull request where you added pylint pragmas because the import order was non-obvious.  Shouldn't those files also be skipped by isort by adding them here?

Reply:
Most of them were under "envs" folder which is covered by the first element in this list. I'm going to rebase and rerun isort again on each folder. If I have to do anything manual I'll add the file to this list.